.github/workflows/deploy.yml

```diff
@@ -11,6 +11,19 @@ jobs:
     strategy:
```

Comment:
Shouldn't this file be called publish?

Reply:
It was stolen from lib-rql)

![image](https://user-images.githubusercontent.com/3644220/181247462-e8c70bdd-73a9-4462-ab2d-9f176a09793a.png)